distros/centos-stream-legacy/centos-stream-legacy.repo

```diff
@@ -0,0 +1 @@
+/usr/share/rpmdistro-repoquery/distros/centos-stream-legacy/centos-stream-legacy.repo
```

Comment:
This file comes from `rpmdistro-repoquery`, right?

*checks https://src.fedoraproject.org/rpms/rpmdistro-repoquery*

So EPEL9 will be required on the systems, OK.

Reply:
Yes. My idea was that we would push off the actual repo configs to something common, rather than maintaining it ourselves. We can also choose to copy them of course.